Assets/MixedRealityToolkit-Examples/Demos/Sharing/BasicAppState/AppStateBasic.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Pixie.StateControl;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Pixie.Demos
```

Comment:
Don't forget to update the namespaces

Reply:
We're going to delay namespace changes until just prior to merge. These scripts live in a separate repo that gets changed frequently in response to internal feedback. In the earlier PR the MRTK namespaces made it tough to maintain parity. (That's partly why the PR was inert for so long.)